src/main/java/com/rakovets/course/java/core/practice/oop/classesandobjects/Point.java

```diff
@@ -0,0 +1,37 @@
+package com.rakovets.course.java.core.practice.oop.classesandobjects;
+import java.awt.geom.Point2D;
+
+public class Point {
+    private double x;
+    public double y;
```

Comment:
Why is the field public?
```suggestion
    private double y;
```